api/par-api.yaml

```diff
@@ -133,6 +133,18 @@ paths:
         schema:
           type: integer
           default: 0
+      - name: ordering
+        in: query
+        description: Used for requested ordered responses, this defines the properties
+          to user for ordering the list of Business Rules returned. 
+          Add 'd-' prefix for descending order.
```

Comment:
you updated the convention, but not the docs :-)

Reply:
whups.  fixed in a new commit.